tests/baselines/reference/api/typescript.d.ts

```diff
@@ -7000,6 +7001,7 @@ declare namespace ts {
         Transient = 33554432,
         Assignment = 67108864,
         ModuleExports = 134217728,
+        All = 67108863,
```

Comment:
```suggestion
        All = -1,
```

Reply:
We should test this, to be certain it doesn't affect something else due to the additional flags that are now included.